AcadeMeData/views.py

```diff
@@ -15,4 +15,4 @@ def register(request):
         form = UserRegistrationForm()
 
     context = {'form': form}
-    return render(request, 'users/register.html', context)
+    return render(request, '../templates/registration/registration.html', context)
```

Comment:
The path changed because 
users/register.html 
is the old form-ui @DanielItkes used to check if his methods work
the new path is the UI style we chose to display.